app/scripts/controllers/swaps.js

```diff
@@ -685,34 +723,114 @@ export default class SwapsController {
 }
 
 /**
- * Calculates the median of a sample of BigNumber values.
+ * Calculates the median overallValueOfQuote of a sample of quotes.
  *
- * @param {import('bignumber.js').BigNumber[]} values - A sample of BigNumber
- * values. The array will be sorted in place.
- * @returns {import('bignumber.js').BigNumber} The median of the sample.
+ * @param {Array} quotes - A sample of quote objects with overallValueOfQuote, ethFee, and metaMaskFeeInEth  properties
+ * @returns {Object} An object with the overallValueOfQuote, ethFee, and metaMaskFeeInEth of the quote with the median overallValueOfQuote
```

Comment:
```suggestion
 * @param {Array} quotes - A sample of quote objects with overallValueOfQuote, ethFee, ethValueOfTokens, and metaMaskFeeInEth  properties
 * @returns {Object} An object with the ethFee, ethValueOfTokens, and metaMaskFeeInEth of the quote with the median overallValueOfQuote
```

The property `ethValueOfTokens` was missing from both of these, and the `overallValueOfQuote` property was mistakenly included in the description of the return (it is not actually present).

Reply:
ethValueOfTokens was added in 9ec3a0d2c and overallValueOfQuote was removed from the return value in 59faf71e1